Tasks/AzureIoTEdgeV2/README.md

```diff
@@ -0,0 +1,101 @@
+# Azure IoT Edge
+
+### Overview
+
+Use this task in a build or release pipeline to build, test, and deploy applications quickly and efficiently to Azure IoT Edge.
+
+### Contact Information
+
+Please report a problem at [Developer Community Forum](https://developercommunity.visualstudio.com/spaces/21/index.html) if you are facing problems in making this task work.  You can also share feedback about the task like, what more functionality should be added to the task, what other tasks you would like to have, at the same place.
```

Comment:
If we are using Developer Community here, how can our team get these feedback in time and response to customers? Just want to fit into your standard process.

Reply:
I'm not sure about that. I just used the same format as [here](https://github.com/iamsrivatsa/azure-pipelines-tasks/tree/srivatsa/IotEdge/Tasks/AzureRmWebAppDeploymentV4) suggested by @pulkitaggarwl and hence we have that section for developer community!

Reply:
I see. I am following up with Pulkit with an email. Thank you.

Reply:
@shizn  DeveloperCommunity is where all our task feedbacks get routed to. 
Lets connect offline on this and figure out how can we loop you in for customer queries. 
Resolving this for now.